docs/packages/pkg/NLopt.rst

```diff
@@ -0,0 +1,21 @@
+.. spelling::
+
+    NLopt
+
+.. index::
+  single: unsorted ; NLopt
+
+.. _pkg.NLopt:
+
+NLopt
+=====
+
+-  `Official <https://official_NLopt>`__
+-  `Hunterized <https://github.com/hunter-packages/NLopt>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/NLopt/CMakeLists.txt>`__
+-  Added by `NLopt_developer <https://github.com/NLopt_developer>`__ (`pr-1617 <https://github.com/ruslo/hunter/pull/1617>`__)
```

Comment:
You should leave your name here. Also you can remove this line if you don't want to do it.